config/locales/en/loans.en.yml

```diff
@@ -165,7 +165,7 @@ en:
       total_fixed_costs: "Total Fixed Costs"
 
     response_set:
-      conflicting_changes: 'Warning: The data has changed since you loaded it. The last update was by %{updater}, %{updated_at}. You may want to check with them to see what they changed or <a href="%{link}" target="_blank">open the latest data in a new tab</a> to compare. You can save your changes anyway, but all other changes since you loaded the page will be lost.'
+      conflicting_changes_html: 'Warning: The data has changed since you loaded it. The last update was by %{updater}, %{updated_at}. You may want to check with them to see what they changed or <a href="%{link}" target="_blank">open the latest data in a new tab</a> to compare. You can save your changes anyway, but all other changes since you loaded the page will be lost.'
```

Comment:
Using `_html` for the key fixes links that get pulled from the yml?

Reply:
yes. html tags actually but this doesn't work on those called from js